R/get_cdc_data.R

```diff
@@ -0,0 +1,41 @@
+#' Retrieve CDC growth chart metadataset
+#'
+#' Retrieves CDC growth chart metadataset
+#'
+#' @param type Type of Chart
+#'
+#' A character is expected amongst: "height", "weight", "bmi"
```

Comment:
What about showing this as:
```
#'   *Permitted Values*: `"height"`, `"weight"`, `"bmi"`
```
I find it easier for users to interpret their options this way.